libsolidity/parsing/Parser.cpp

```diff
@@ -994,11 +994,19 @@ ASTPointer<UsingForDirective> Parser::parseUsingDirective()
 				Token operator_ = m_scanner->currentToken();
 				if (!util::contains(userDefinableOperators, operator_))
 				{
+					auto operatorName = [&]() -> string {
+						if (!m_scanner->currentLiteral().empty())
+							return m_scanner->currentLiteral();
+						else if (char const* tokenString = TokenTraits::toString(operator_))
+							return string(tokenString);
+						return {};
+					};
```

Comment:
This lambda is used exactly once and does not take parameters. It would be simpler to replace it with an intermediate variable.

Reply:
That's true.